tutorials/heritage/index.md

```diff
@@ -0,0 +1,6 @@
+\defgroup tutorial_heritage Heritage tutorials
+\ingroup Tutorials
+\brief Tutorials not anymore in line with times
```

Comment:
What is "in line with times"?

What about "Legacy ROOT tutorials."

Reply:
OK. It sounded more sophisticated but got tricked by this nuance not present in latin languages :-P